### DISCUSSION TITLE
Alembic migrations for Django users

### DISCUSSION THREAD

**Discussion by abitrolly** [sqlalchemy/alembic#970] [Ideas]
In https://github.com/pypa/warehouse/pull/9512/files#diff-b912d97d6c38546cdc81c0c2cc35833412c3b8ccd46a62ac5767f030a35bef73 PyPI project had a contribution with a piece of technical debt, because there is not chapter like "Alembic migration for Django users" in Alembic docs and the PR was already too big.

**Comment by CaselIT** [comment]
Hi,

I'm not sure what's the question. You are proposing adding a page that explain how to use alembic in a django project?
Could you be more specific?
Thanks

**Reply by abitrolly** [reply]
Adding a page that shows how writing migration for Alembic is different from writing migration for Django. When people who already know Django need to contribute to Alembic backed project and don't have much time to learn new framework from scratch.

**Reply by zzzeek** [reply]
this sounds like something more appropriate for a blog post done by someone with experience in this area.  we know nothing about Django here.

**Reply by zzzeek** [reply]
like this one for example: https://blog.dotmodus.com/maintaining-sqlalchemy-models-like-django-orm-with-alembic/

**Reply by zzzeek** [reply]
put another way.  does django's documentation have guides for people coming from alembic?  django has orders of magnitude more developer resources than we do, were they able to achieve this ?

**Reply by abitrolly** [reply]
Well, as a DevOps I had to deal with a lot of stuff, including Alembic, Django, Ruby on Rails migrations, but right now I don't have a Django project to refresh my knowledge. If there are any sponsors who are interested to grow the user base for Alembic (so that they can more easily hire people who are able to deal with it), I would contact them to hire an ORM expert, who explored different frameworks to know the difference and write the stuff.
